src/oncall/ui/static/js/oncall.js

```diff
@@ -579,11 +579,11 @@ var oncall = {
               },
               footer: function(resp){
                 if (teamsCt > typeaheadLimit) {
-                  return '<div class="tt-see-all"><a href="/query/' + resp.query + '/teams" data-navigo> See all ' + teamsCt + ' results for teams »</a></div>';
+                  return '<div class="tt-see-all"><a href="/query/' + Handlebars.escapeExpression(resp.query) + '/teams" data-navigo> See all ' + teamsCt + ' results for teams »</a></div>';
```

Comment:
Like Colin mentioned, for places where resp.query is being interpolated into a URL, you should probably wrap it in encodeURIComponent() as well, to ensure URL-specific characters are escaped into percent notation, generating a working URL. As in:

```
'<div class="tt-see-all"><a href="/query/' + Handlebars.escapeExpression(encodeURIComponent(resp.query)) + ' etc etc '
```

Reply:
added